core-common/src/main/java/org/zalando/nakadi/repository/kafka/KafkaFactory.java

```diff
@@ -151,6 +157,10 @@ public void terminateProducer(final Producer<byte[], byte[]> producer) {
     }
 
     public Consumer<byte[], byte[]> getConsumer(final String clientId) {
+
+        return getConsumer();
```

Comment:
First we want to see the rate before introducing the pool — will change it back in the next PR.